db/migrate/20220809145455_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.string :Bio
+      t.integer :PostsCounter
```

Comment:
- [ ] please it would be better to follow the convention of creating columns in the database they should be in snake_case like that: comment_counter ...etc please try to do that